### PR TITLE
Update index.md for the JavaScript Language Overview

### PR DESCRIPTION
**Pull request by ssiano**
### Description
Minor edits

### CODE REVIEW

File: files/en-us/web/javascript/language_overview/index.md [hunk 1]
```diff
@@ -748,7 +748,7 @@ There are three idiomatic ways to write asynchronous code in JavaScript:
 - [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)-based
 - [`async`](/en-US/docs/Web/JavaScript/Reference/Statements/async_function)/[`await`](/en-US/docs/Web/JavaScript/Reference/Operators/await), which is a syntactic sugar for Promises
 
-For example, here's how a file-read operation may look like in JavaScript:
+For example, here's how a file-read operation may look in JavaScript:
```
Comment (hamishwillee) [inline files/en-us/web/javascript/language_overview/index.md:751]:
```suggestion
For example, here's what a file-read operation might look like in JavaScript:
```

**Review by hamishwillee — APPROVED** [review]
Looks good thanks - and welcome to MDN!

**Comment by ssiano** [comment]
Thanks @hamishwillee for providing such fantastic docs!

**Comment by hamishwillee** [comment]
You're welcome "on behalf of MDN" :-)

**Pull request merged**